addons/mail/static/src/components/emoji_grid_view/emoji_grid_view.js

```diff
@@ -15,6 +15,7 @@ export class EmojiGridView extends Component {
         onMounted(() => {
             this.emojiGridView.calculateDimensions();
         });
+        onWillStart(() => this.messaging.emojiRegistry.loadEmojiData());
```

Comment:
Problem with lazy loading when clicking on emoji picker is that user will see long insertion of emoji while opening the emoji picker, which is not pretty.

I would do the following improvements:
- have spinner while `isLoaded` is falsy
- lazy-load much earlier, even when not actually using the emoji picker

1st part will avoid seeing the insertion of emoji while using the emoji picker. All features will behave correctly when all emojis are loaded, such as search.
2nd part will make showing of emoji picker fast. With current lazy-load at opening of emoji picker, this is slow. Better UX is to lazy-load sooner, even if we don't actually use the emoji picker. Could do this in `EmojiRegistry/_created`.